src/inspector_agent.cc

```diff
@@ -567,12 +578,30 @@ void AgentImpl::WaitForDisconnect() {
                            v8::ReadOnly).FromJust();                          \
   } while (0)
 
+void AddCommandLineAPIMethod(const v8::FunctionCallbackInfo<v8::Value>& info) {
```

Comment:
@addaleax & @bnoordhuis: Thanks a lot for the explanation. I'm pretty sure I learned & forgot this before. Once or twice.

I updated this function and also made the other two exposed C++ methods (`InspectorConsoleCall`/`InspectorWrapConsoleCall`) static so the style is consistent inside of the file. Went with `static` over `inline` because it seemed to be the dominant style in other `*.cc` files.